.eslintrc.js

```diff
@@ -56,6 +59,10 @@ module.exports = {
 		],
 		'@typescript-eslint/no-explicit-any': 'off',
 		'@typescript-eslint/no-var-requires': 'off',
+		'@typescript-eslint/ban-types': 'off',
```

Comment:
this is not needed.

Reply:
I get quite a few errors when this is taken out:

`Users/vuyani/Documents/sanbi/portal/global/hooks/useMuseData/index.ts
  74:15  error  Don't use `Function` as a type. The `Function` type accepts any function-like value.
It provides no type safety when calling the function, which can be a common source of bugs.
It also accepts things like class declarations, which will throw at runtime as they will not be called with `new`.
If you are expecting the function to accept certain arguments, you should explicitly define the function shape  @typescript-eslint/ban-types

/Users/vuyani/Documents/sanbi/portal/global/utils/pages/index.tsx
  30:13  error  Don't use `{}` as a type. `{}` actually means "any non-nullish value".
- If you want a type meaning "any object", you probably want `object` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.
- If you want a type meaning "empty object", you probably want `Record<string, never>` instead  @typescript-eslint/ban-types

/Users/vuyani/Documents/sanbi/portal/global/utils/processStream.ts
  28:21  error  Don't use `Function` as a type. The `Function` type accepts any function-like value.
It provides no type safety when calling the function, which can be a common source of bugs.
It also accepts things like class declarations, which will throw at runtime as they will not be called with `new`.
If you are expecting the function to accept certain arguments, you should explicitly define the function shape  @typescript-eslint/ban-types

✖ 3 problems (3 errors, 0 warnings)

vuyani@Vuyanis-MacBook-Pro portal % npm run lint

> virusseq-dataportal-ui@1.14.0 lint
> eslint --ext .ts,.tsx '.'

vuyani@Vuyanis-MacBook-Pro portal % npm run lint

> virusseq-dataportal-ui@1.14.0 lint
> eslint --ext .ts,.tsx '.'


/Users/vuyani/Documents/sanbi/portal/global/hooks/useMuseData/index.ts
  74:15  error  Don't use `Function` as a type. The `Function` type accepts any function-like value.
It provides no type safety when calling the function, which can be a common source of bugs.
It also accepts things like class declarations, which will throw at runtime as they will not be called with `new`.
If you are expecting the function to accept certain arguments, you should explicitly define the function shape  @typescript-eslint/ban-types

/Users/vuyani/Documents/sanbi/portal/global/utils/pages/index.tsx
  30:13  error  Don't use `{}` as a type. `{}` actually means "any non-nullish value".
- If you want a type meaning "any object", you probably want `object` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.
- If you want a type meaning "empty object", you probably want `Record<string, never>` instead  @typescript-eslint/ban-types

/Users/vuyani/Documents/sanbi/portal/global/utils/processStream.ts
  28:21  error  Don't use `Function` as a type. The `Function` type accepts any function-like value.
It provides no type safety when calling the function, which can be a common source of bugs.
It also accepts things like class declarations, which will throw at runtime as they will not be called with `new`.
If you are expecting the function to accept certain arguments, you should explicitly define the function shape  @typescript-eslint/ban-types

✖ 3 problems (3 errors, 0 warnings)`

Reply:
Ok odd.. I didn't get these errors. Well leave it in for now. I presume that these errors are a result of using placeholder code.